cgi-bin/CATS/UI/ProblemHistory.pm

```diff
@@ -89,6 +89,8 @@ sub problem_history_tree_frame {
     $is_jury or return;
     my $pr = _get_problem_info($p) or return $p->redirect(url_f 'contests');
 
+    my $log = problem_history_delete_frame($p, $pr) if $p->{file_name};
```

Comment:
`my` ... `if` is deprecated. Use my = ? : undef;
However, here it is better to just
`if (my $log = ...) { $t->param(problem_import_log ...`